### DISCUSSION TITLE
Calculation power utilization in rack

### DISCUSSION THREAD

**Discussion by strelok00** [netbox-community/netbox#11774] [Help Wanted!]
Hi, I have NetBox 3.16 version
When I adding second PDU in my rack I see that power utilization is divided between both PDUs.
I tried make second PDU as redundant Role but nothing changes

PDU1 (10 000VA) + PDU2 (10 000VA) must be taken into account as 10 000VA and utilization usage respectively.

Is it possible to change this calculation ? ?

**Comment by jeremystretch** [comment]
> Hi, I have NetBox 3.16 version

Did you mean v3.1.6? If so, that's a bit out of date and I'd suggest upgrading to the latest release.
